### PR TITLE
Captions and transcripts

### LINKED ISSUE #8292: Exhibition - guides: "Captions and transcripts" page UI adjustments
As a Wellcome Visitor I would like to be able to quickly identify the correct section of the captions and transcripts to see the relevant information for an item in the gallery

Design: https://zpl.io/406Y3Ep

### PR DESCRIPTION
**Pull request by gestchild**
Fixes #8292

screencapture-localhost-3002-guides-exhibitions-YvJ4UhAAAPgZztMl-captions-and-transcripts-2022-08-24-16_37_19

### CODE REVIEW

File: content/webapp/pages/exhibition-guide.tsx [hunk 1]
```diff
@@ -547,7 +553,7 @@ const ExhibitionGuidePage: FC<Props> = props => {
           >
             <Layout8 shift={false}>
               <Space v={{ size: 'l', properties: ['margin-bottom'] }}>
-                <h1 className="h1">Other exhibition guides available</h1>
+                <h2 className="h2">Other exhibition guides available</h2>
```
Comment (melanierogan) [inline content/webapp/pages/exhibition-guide.tsx:556]: Cheers for reverting, I was attempting to make it as per font size in Zeplin but then realised the ones in the design aren't mapped to any of the values we use.

File: content/webapp/components/ExhibitionCaptions/ExhibitionCaptions.tsx [hunk 2]
```diff
@@ -87,70 +159,139 @@ const Stop: FC<{ stop: Stop }> = ({ stop }) => {
...
+          <Space
+            v={{
+              size: 'xl',
+              properties: ['margin-bottom'],
+            }}
+          >
+            <Divider color={`pumice`} isKeyline={true} />
+          </Space>
+          <div className="flex flex--wrap">
+            <Tombstone />
```
Comment (melanierogan) [inline content/webapp/components/ExhibitionCaptions/ExhibitionCaptions.tsx:179]: 👍

File: content/webapp/components/ExhibitionCaptions/ExhibitionCaptions.tsx [hunk 3]
```diff
@@ -87,70 +159,139 @@ const Stop: FC<{ stop: Stop }> = ({ stop }) => {
...
+                {number ? `${number}. ` : ''}
+                {!hasContext && title}
+              </TombstoneTitle>
+              <div className={font('intr', 4)}>
+                <PrismicHtmlBlock html={tombstone} />
+              </div>
+            </Tombstone>
+
+            <CaptionTranscription>
+              {isFirstStop && hasContext && title.length > 0 && (
```
Comment (melanierogan) [inline content/webapp/components/ExhibitionCaptions/ExhibitionCaptions.tsx:219]: Nice and clear

**Review by melanierogan — APPROVED** [review]
Really lovely piece of work

File: content/webapp/components/ExhibitionCaptions/ExhibitionCaptions.tsx [hunk 4]
```diff
@@ -87,70 +176,139 @@ const Stop: FC<{ stop: Stop }> = ({ stop }) => {
...
-                    : 'Read full transcript'
-                }
-              />
-            )}
-          </Transcription>
-        )}
-      </CaptionTranscription>
-    </Space>
+        >
+          <div className="flex flex--wrap container">
```
Comment (davidpmccormick) [inline content/webapp/components/ExhibitionCaptions/ExhibitionCaptions.tsx:224]: There's something about having `flex` and `container` on the same element that is setting off a distant alarm bell for me, but looking at what `.container` does, I'm not sure why now – I've probably misremembered something

File: content/webapp/pages/exhibition-guide.tsx [hunk 5]
```diff
@@ -316,17 +321,14 @@ const Stops: FC<StopsProps> = ({ stops, type }) => {
 };
 
 const ExhibitionStops: FC<StopsProps> = ({ stops, type }) => {
+  const numberedStops = stops.filter(c => c.number);
```
Comment (davidpmccormick) [inline content/webapp/pages/exhibition-guide.tsx:324]: Is this not in `main` already?

**Review by davidpmccormick — APPROVED** [review]

**Pull request merged**